### DISCUSSION TITLE
关于最高法院245号判决，开发商违约，购房者不承担烂尾楼还贷责任的问题

### DISCUSSION THREAD

**Discussion by KisaraKira** [WeNeedHome/SummaryOfLoanSuspension#459] [General]
理论上这种烂尾房是不用再偿还银行贷款的，最高法19年做过一个经典判决，那么为啥还有人说烂尾楼断供会上征信呢？

**Comment by hash070** [comment]
![9ADB88BA-B264-4DE0-84FB-668978EA7B6F](https://user-images.githubusercontent.com/62081013/179148262-14868057-cfdb-43bd-8b15-e5bd40569b67.jpeg)

**Comment by dalezhang93** [comment]
不做核酸都能上征信，断供妥妥的给上征信  > 理论上这种烂尾房是不用再偿还银行贷款的，最高法19年做过一个经典判决，那么为啥还有人说烂尾楼断供会上征信呢？

**Comment by yalo-ddaa** [comment]
因为我国不是判例法，别的法院的判决不一定会被法官采纳。
所以除非最高法院出一个司法解释，否则其他地方的法院依然可以我行我素。

**Reply by KisaraKira** [reply]
原来如此，学习了

**Comment by KisaraKira** [comment]
刚查了一下，中国确实不是判例法而是成文法，因此这里提到的245号判决并不一定能成为断供者的挡箭牌
> 中国系属大陆法系，以成文法为主要法源。法律规则是由立法机关制定，法官主要依赖法律规则，运用三段论模式审理案件，以达规范人际生活关系，协调各种法律利益，维护社会正义及秩序的法律目的。
